script/objects.js

```diff
@@ -0,0 +1,202 @@
+const car = {
+    maker: "Renault Fluence",
+    model: "Megane Sedan",
+    yearManufacture: 2021,
+    averageSpeed: 95,
+    fuelTank: 50,
+    "average fuel consumption per 100 km": 4.4,
```

Comment:
This key prop is hard to maintain, no need for such detailed info about the object. Developers will understand simpler form, like

`averageCons: 4.4` for example